src/docs/development/platform-integration/add-to-app/performance.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Load sequence, performance and memory
+---
+
+This page serves to inform the reader of the breakdown of the steps involved
+to show a Flutter UI. Knowing this, the user can make better informed decisions
+about when to pre-warm the Flutter engine, which operations are possible
+at which stage and the latency and memory costs of those operations.
+
+## Loading Flutter
+
+Both Android/iOS (the 2 supported platforms for integrating into existing
```

Comment:
You say "Both", but then list 3 options, unless I am misunderstanding. How about:

"Android and iOS apps (the two supported platforms for integrating into existing apps),
full Flutter apps, and add-to-app patterns have a similar sequence of conceptual 
loading steps when displaying Flutter UI."

Reply:
I meant both dimensions / duos. But your version is better.